sdk/formrecognizer/test-resources.json

```diff
@@ -73,7 +73,7 @@
         },
         "selectionMarkTrainingDataContainer": {
             "type": "string",
-            "defaultValue": "selectionmark"
+            "defaultValue": "formselectionmark"
```

Comment:
Was this changed or we had the incorrect one before?

Reply:
the folder name changed